### PR TITLE
[JavaScript] Add lint tools

### PR DESCRIPTION
**Pull request by bytemain**
## What do these changes do?

Add eslint

## Check code requirements

- [x] tests added / passed (if needed)
- [x] Ensure all linting tests pass, see [here](https://github.com/alipay/fury/blob/main/CONTRIBUTING.rst) for how to run them

### CODE REVIEW

File: javascript/.eslintrc.cjs [hunk 1]
```diff
@@ -1,12 +1,23 @@
+const stylistic = require("@stylistic/eslint-plugin");
+
+const customized = stylistic.configs.customize({
+  indent: 2,
```
Comment (bytemain) [inline javascript/.eslintrc.cjs:4]: Just use the default value, if you have any idea, please say.

**Comment by chaokunyang** [comment]
This is great, could we add code lint to code style check CI:
```yml
  lint:
    name: Code Style Check
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - name: Check License Header
        uses: korandoru/hawkeye@v3
      - name: Set up Python 3.8
        uses: actions/setup-python@v2
        with:
          python-version: 3.8
      - name: Check code style
        run: ./ci/run_ci.sh format
```

File: javascript/.eslintrc.cjs (hunk 1 above)
Comment (chaokunyang) [reply javascript/.eslintrc.cjs:4]: indent with 2 spaced looks good to me, we can write and read more code in one line.

**Review by chaokunyang — APPROVED** [review]
LGTM

File: ci/run_ci.sh [hunk 2]
```diff
@@ -253,6 +253,9 @@ case $1 in
       cd "$ROOT/java"
       mvn -T10 -B --no-transfer-progress spotless:check
       mvn -T10 -B --no-transfer-progress checkstyle:check
+      pushd "$ROOT/javascript"
```
Comment (wangweipeng2) [inline ci/run_ci.sh:256]: There is already a checker in ci/format.sh

Comment (bytemain) [reply ci/run_ci.sh:256]: Yes, I will submit a commit remove this redundant check

**Pull request merged**